### PR TITLE
libatomic_ops Recipe

### PR DESCRIPTION
**Pull request by stefan-balke**
Hopefully my last dependency.

### CODE REVIEW

File: recipes/libatomic_ops/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+
+test:
+  commands:
+    - true
```
Comment (ocefpaf) [inline recipes/libatomic_ops/meta.yaml]: I am not comfortable with this. Can we test for a library existence? See the recipe below for an example:

https://github.com/conda-forge/staged-recipes/pull/239/files#diff-efc5664ac8f8d3a512d291b45c29608dR29

**Comment by ocefpaf** [comment]
Thanks for persevering! This looks great. I guess that know you are so experienced in creating conda recipes for conda-forge that you can start commenting someone elses PRs :wink:

**Comment by ocefpaf** [comment]
`libatomic_ops` has an underscore and we need a new conda-smithy release :unamused: to deal with that. The feedstock was not create. See https://github.com/conda-forge/libatomic_ops-feedstock


```shell
/home/travis/build/conda-forge/conda-forge.github.io/scripts/update_feedstocks_submodules.py:60: UserWarning: Unable to add the submodule libatomic_ops. This is likely because the repo has no commits, which likely means something went wrong with feedstock generation. Will skip adding this submodule and continue.
  "Will skip adding this submodule and continue.".format(feedstock.package_name)
```

Ping @pelson. We can wait, but if you are too busy I can try to make the release.

**Comment by pelson** [comment]
> Ping @pelson. We can wait, but if you are too busy I can try to make the release.

Solved by v0.7.1, right?

**Comment by jakirkham** [comment]
Yeah, this was before the release.

**Comment by ocefpaf** [comment]
> Yeah, this was before the release.

libatomic_ops was *there and back again* :wink:

**Pull request merged**